lib/chess/auth/error_handler.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Chess.Auth.ErrorHandler do
+  @moduledoc false
+
+  import Plug.Conn
+
+  def auth_error(conn, {_type, _reason}, _opts) do
+    conn
+    |> Phoenix.Controller.put_flash(:info, "You must be logged in")
```

Comment:
Should I be directly accessing `Phoenix.Controller` functions here or is there a better way?

Reply:
If you import `Phoenix.Controller`, or maybe `use Chess.Web, :controller` if you have one, then it'll feel more like a regular controller where you can just call `put_flash`. It's really just up to you.